AdChoicesManager.js

```diff
@@ -1,83 +1,84 @@
 // @flow
-import React, {PropTypes} from 'react';
+import React, { PropTypes } from 'react';
```

Comment:
Are PropTypes used in this component? It's not related to this PR but we should get rid of it so that's why i am asking.

Reply:
Unrelated to this PR, but yeah there are some Flow and ESLint errors that should be fixed (unused imports, broken types and such)